src/aff_expr.jl

```diff
@@ -16,12 +16,13 @@
 #############################################################################
 
 # Utilities for OrderedDict
-function _add_or_set!(dict::OrderedDict{K,V}, k::K, v::V) where {K,V}
-    # TODO: This unnecessarily requires two lookups for k.
+function _add_or_set!(exprterms::OrderedDict{K,V}, k::K, v::V) where {K,V}
```

Comment:
Either `expr_terms` (http://www.juliaopt.org/JuMP.jl/v0.19.0/style/#Use-of-underscores-within-names-1) or leave as `dict` .